### DISCUSSION TITLE
How to get vendorId and vendorGroupId mapping information?

### DISCUSSION THREAD

**Discussion by Sandeep0393** [amzn/selling-partner-api-models#3219] [General]
Hi,
We are using `sellerId` to relate selling partner API data with Advertising API data for a seller but  we are not able to do the same for vendors because when a seller authorizes Selling Partner API, Amazon returns `sellerId` but for vendor it returns `vendorGroupId ` as `selling_partner_id` value where as Amazon Advertising API returns `sellerId`/`vendorId ` via get profiles API call.  

So, we don't have `vendorId` and `vendorGroupId` mapping information. Is there any possibility to fetch such information via API?

Thanks!

**Comment by Guerri114** [comment]
@Sandeep0393 : I don't think there is a clean mapping between vendor central (retail) accounts and vendor advertising accounts like there is for sellers. I think you will need to build this mapping yourself, based off of input from your client.
